src/Compilers/Core/Portable/Generated/FlowAnalysis.Generated.cs

```diff
@@ -108,4 +109,5 @@ public interface IFlowAnonymousFunctionOperation : IOperation
         /// </summary>
         IMethodSymbol Symbol { get; }
     }
+    #endregion
```

Comment:
you appear to be wrapping the entire namespace.  does this region serve any purpose?

Reply:
Its generated. I don't feel it's worth special casing in the generator.